frontend/src/App.js

```diff
@@ -1,15 +1,26 @@
-
 import './App.css';
+import React, { useState } from 'react';
+import Navigation from './Navigation';
+import StepTime from './StepTime';
+import Chart from './Chart';
+import Graph from './Graph';
+import useStepTime from './useStepTime';
 
 function App() {
+  const [currentView, setCurrentView] = useState('step_time');
+  const stepTime = useStepTime();
+
+  const views = {
+    step_time: <StepTime stepTime={stepTime} />,
+    chart: <Chart stepTime={stepTime} />,
+    graph: <Graph stepTime={stepTime} />,
+  };
+
```

Comment:
**Just a note**
It is out of the scope of your current PR but going forward, we will probably not render components directly from `App.js`. This would probably just contain routing. Let's discuss this and see what works best for our project.

@Nissa9902 @Nissa1218 @JackMcPhillips1543 Take note. In our next meeting, we need to establish an acceptable directory/file format. Basically "what goes in where"